SwiftSampleProject/SwiftyVokoder/Models/Station.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Station.swift
+//  
+//
+//  Created by Carl Hill-Popper on 11/13/15.
+//  Copyright © 2015 Vokal.
+//
+
+import Foundation
+import CoreData
+import CoreLocation
+import Vokoder
+
+//lets pretend we're using mogenerator to create attribute constants
+enum StationAttributes: String {
+    case name
+    case longitude
+    case locationString
+    case latitude
+    case identifier
+    case descriptiveName
+    case accessible
+}
+
+@objc(Station)
```

Comment:
That's good to know but I don't think that's specific to Vokoder.  I don't know if it's worth including the README, but maybe as a blog post about Swift/Obj-C intro gotchas?